### ISSUE TITLE
dataset and model

### ISSUE THREAD

**Issue by PoonamZ** [iwantooxxoox/keras-facenet#12]
@iwantooxxoox  I would like to know about the dataset structure used to train the model nn4.small2.h5. 
Can you please elaborate the label file used while training? how we will get the 128x1 vector ?

**Comment by Tornquist** [comment]
@PoonamZ The model isn't trained here. Look at https://cmusatyalab.github.io/openface/ to see the logic behind building the model. This is just a conversion of the model from Torch to Keras.

**Comment by PoonamZ** [comment]
OK thanx.
